admin/services/data-access/pupil-status-code.data.service.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const sqlService = require('./sql.service')
+
+const pupilStatusCodeDataService = {}
+const table = '[pupilStatusCode]'
+
+/**
+ * Return a list of pupil status codes
+ * @return {Promise<results>}
+ */
+pupilStatusCodeDataService.sqlFindStatusCodes = async () => {
+  // TODO: data-refactor: candidate for caching
+  const sql = `
+  SELECT 
+    id, 
+    code, 
+    statusDesc 
+  FROM ${sqlService.adminSchema}.${table} 
+  ORDER BY statusDesc ASC`
+  return sqlService.query(sql)
```

Comment:
doesn't query take two args?
You might have to pass an empty array as second for parameters

Reply:
The second param to query takes a default option `[]` so can be safely ignored.

Reply:
yea noticed